src/components/PlioQuestion.vue

```diff
@@ -149,6 +150,10 @@ export default {
       // Wait 200 ms because it takes some time to find the DOM elements
       if (this.plioQuestion.state == "answered") {
         setTimeout(() => {
+          // highlight wrong/right depending on what the user answered in previous session
+          document.getElementById(this.plioQuestion.user_answer).checked = true
+          this.selectedOption = this.plioQuestion.user_answer
+          this.checkAnswer()
```

Comment:
When re-visiting a question that was already answered in an older session, these steps need to followed:
1. Check the radio button corresponding to the option which was answered in the older session
2. set that value in variable `selectedOption`
3. Call `checkAnswer` method so it can detect weather the answer was correct/not and can highlight with colors accordingly